crypto/src/lib.rs

```diff
@@ -215,7 +209,6 @@ impl KeyPair {
     ///
     /// # Errors
     /// Fails if decoding fails
-    #[cfg(any(feature = "std", feature = "ffi_import"))]
     pub fn generate_with_configuration(configuration: KeyGenConfiguration) -> Result<Self, Error> {
```

Comment:
Similar concerns here as with `generate`, though `KeyGenOption::UseSeed` or `KeyGenOption::FromPrivateKey` are deterministic

Reply:
Removed `Option<KeyGenOption>` and added feature-gated `KeyGenOption::Random`